src/NUnitTestAdapter/NavigationData.cs

```diff
@@ -0,0 +1,31 @@
+﻿// ****************************************************************
+// Copyright (c) 2016 NUnit Software. All rights reserved.
+// ****************************************************************
```

Comment:
The copyright header is the one I used originally for the VS Adapter and has never been changed.

Note that it allows releasing the software under multiple licenses, which was my original intent. I planned to have both GPL and proprietary licenses available. As far as I remember, I changed my mind and used MIT before the initial release.

We may need to have an issue somewhere (where?) to discuss this globally. Although many projects do include the text of short licenses like BSD and MIT at the top of each file, not all of them do. So I would be OK with a shorter header. However, we need to have a clear understanding of what goes into the header (as well as the license) for each separate project we manage.

Should this perhaps be a docs issue?